core/src/main/scala/org/http4s/StaticFile.scala

```diff
@@ -87,11 +88,18 @@ object StaticFile {
             else `Transfer-Encoding`(TransferCoding.chunked)
           val headers = Headers(lenHeader :: lastModHeader ::: contentType)
 
-          Some(
-            Response(
-              headers = headers,
-              body = readInputStream[F](F.delay(url.openStream), DefaultBufferSize, blocker)
-            ))
+          Try(url.openStream()).fold(
```

Comment:
I wonder if we really need `Try` here since we already have `F: Sync` around and can suspend effects in it and do all necessary error handling inside?

Reply:
Fixed